bluemira/radiation_transport/radiation_tools.py

```diff
@@ -110,6 +109,11 @@ def upstream_temperature(
     -------
     t_upstream_kev:
         upstream temperature. Unit [keV]
+
+    Notes
+    -----
+    Reference:
+    Pitcher et al., Plasma Physics and Controlled Fusion 39.6 (1997): 779
```

Comment:
Could we use the doi directive or role eg:

directive:
https://github.com/Fusion-Power-Plant-Framework/bluemira/blob/9c4ebf72f13469cd174a51d17bfc5a7bd45bdab4/bluemira/plasma_physics/rules_of_thumb.py#L84-L85

or role

https://github.com/Fusion-Power-Plant-Framework/bluemira/blob/9c4ebf72f13469cd174a51d17bfc5a7bd45bdab4/bluemira/plasma_physics/scaling_laws.py#L168

Reply:
done.